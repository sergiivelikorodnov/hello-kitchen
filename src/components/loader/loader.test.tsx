import React from 'react'
import { render } from '@testing-library/react'
import Loader from './loader'

describe('Render Loading Component', () => {
  it('should render correctly', () => {
    const { container } = render(<Loader />)
    expect(container).toMatchSnapshot()
  })
})
