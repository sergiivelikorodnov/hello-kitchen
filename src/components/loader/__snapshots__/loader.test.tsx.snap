// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Loading Component should render correctly 1`] = `
<div>
  <div
    data-testid="loading"
    style="display: flex; justify-content: center; margin-top: 40px;"
  >
    <div
      data-testid="tail-spin-loading"
      style="display: flex;"
    >
      <svg
        aria-label="loading"
        data-testid="tail-spin-svg"
        height="50"
        viewBox="0 0 38 38"
        width="50"
        xmlns="http://www.w3.org/2000/svg"
      >
        <defs>
          <lineargradient
            id="a"
            x1="8.042%"
            x2="65.682%"
            y1="0%"
            y2="23.865%"
          >
            <stop
              offset="0%"
              stop-color="grey"
              stop-opacity="0"
            />
            <stop
              offset="63.146%"
              stop-color="grey"
              stop-opacity=".631"
            />
            <stop
              offset="100%"
              stop-color="grey"
            />
          </lineargradient>
        </defs>
        <g
          fill="none"
          fill-rule="evenodd"
        >
          <g
            transform="translate(1 1)"
          >
            <path
              d="M36 18c0-9.94-8.06-18-18-18"
              id="Oval-2"
              stroke="grey"
              stroke-width="2"
            >
              <animatetransform
                attributeName="transform"
                dur="0.9s"
                from="0 18 18"
                repeatCount="indefinite"
                to="360 18 18"
                type="rotate"
              />
            </path>
            <circle
              cx="36"
              cy="18"
              fill="#fff"
              r="1"
            >
              <animatetransform
                attributeName="transform"
                dur="0.9s"
                from="0 18 18"
                repeatCount="indefinite"
                to="360 18 18"
                type="rotate"
              />
            </circle>
          </g>
        </g>
      </svg>
    </div>
  </div>
</div>
`;
